### PR TITLE
Chinese Translation Completion

### PR DESCRIPTION
**Pull request by zero6six**

### CODE REVIEW

File: nomi-labs/assets/nomilabs/lang/zh_cn.lang [hunk 1]
```diff
@@ -489,43 +455,95 @@ tooltip.nomilabs.hand_framing_tool.front=前面: %s
...
-tooltip.nomilabs.universal_crystallizer.description=An immense device, capable of turning raw materials into complex matters.
-tooltip.nomilabs.dme_sim_chamber.description=A high powered simulation device, capable of performing more complex simulations faster than ever.
-tooltip.nomilabs.greenhouse.description=A high powered growing chamber, capable of mass-growing even the most incomprehensible plants.
\ No newline at end of file
+tooltip.nomilabs.actualization_chamber.description=一台强大的机器，可以利用最复杂的物质并将其转化为原材料。
+tooltip.nomilabs.universal_crystallizer.description=一种巨大的装置，能够将原材料转化为复杂物质。
+tooltip.nomilabs.dme_sim_chamber.description=一种高功率模拟设备，能够比以往更快地执行更复杂的模拟。
+tooltip.nomilabs.greenhouse.description=一个高功率的生长室，即使是最难以理解的植物也能大规模生长。
+
+
```
Comment (IntegerLimit) [inline nomi-labs/assets/nomilabs/lang/zh_cn.lang]: Yep, all entries below this have been REMOVED, not missing. can you delete them please?

Thank you for doing this though, but just know that you can trust the english langs in this repository, they are auto-generated from the english langs in the original repository.

If you do feel like something might be missing, what you have done here is a great way of bringing that to our attention, but you could also ping me in the translations thread.

Apart from this, its all good.

File: nomi-ceu/assets/contenttweaker/lang/zh_cn.lang
Comment (IntegerLimit) [inline nomi-ceu/assets/contenttweaker/lang/zh_cn.lang:1]: Also this file is not needed. Please delete it.

**Review by IntegerLimit — CHANGES_REQUESTED** [review]

File: nomi-ceu/assets/contenttweaker/lang/zh_cn.lang
Comment (IntegerLimit) [reply nomi-ceu/assets/contenttweaker/lang/zh_cn.lang:1]: Sorry, I did not realise that the english version was autogenerated by the nomi-ceu syncing. This is fine then.

**Review by IntegerLimit — CHANGES_REQUESTED** [review]
Can you also add the changes in https://github.com/Nomi-CEu/Nomi-CEu-Translations/commit/0c871386a376f93ae87b055dc06653cf8aaf37f2 first?

File: nomi-labs/assets/nomilabs/lang/zh_cn.lang (hunk 1 above)
Comment (zero6six) [reply nomi-labs/assets/nomilabs/lang/zh_cn.lang]: Okay, I finished.

**Comment by zero6six** [comment]
> Can you also add the changes in [0c87138](https://github.com/Nomi-CEu/Nomi-CEu-Translations/commit/0c871386a376f93ae87b055dc06653cf8aaf37f2) first?

Finished. But using `git rebase` seemed to cause a force push.

**Comment by zero6six** [comment]
By the way, because of the final exam, I may not be online very often recently. I'm sorry about that.

**Review by IntegerLimit — APPROVED** [review]

**Pull request merged**